src/commerce-handler.js

```diff
@@ -1,66 +1,70 @@
+var eventMapping = {
```

Comment:
The items capitalized are Branch standard events. The others are not standard, but custom. They could be capitalized but it causes confusion with customers on our dashboard because standard events are always displayed in capitals.